src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -692,7 +686,13 @@ class ActivityResourceIntTest : IntegrationTestBase() {
               courseScheduleRule()
             }
           }
-          courseAllocation(offenderBooking, courseActivity) { payBand() }
+          offender = offender(nomsId = "A1234TT") {
+            booking(agencyLocationId = "LEI") {
+              incentive(iepLevelCode = "STD")
+              courseAllocation(courseActivity) { payBand() }
+            }
+          }
+          offenderBooking = offender.latestBooking()
```

Comment:
All tests have been changed to the DSL usage of:
* create the program service, course activity and schedule
* create the offender and booking, allocating the booking to the course activity